models.py

```diff
@@ -590,12 +590,16 @@ def deploy_to_path(self, path=None):
         if use_path:
             self._install_nni_flows(use_path)
             self._install_uni_flows(use_path)
-            self.activate()
-            self.current_path = use_path
-            self.sync()
-            log.info(f"{self} was deployed.")
-            return True
-        return False
+        elif self.uni_a.interface.switch == self.uni_z.interface.switch:
+            self._install_direct_uni_flows()
+            use_path = Path()
+        else:
+            return False
+        self.activate()
+        self.current_path = use_path
+        self.sync()
+        log.info(f"{self} was deployed.")
+        return True
```

Comment:
Should this code really be here or is it from another branch/PR?

Reply:
I don't think so, should I just drop this commit?